src/gui/findReplace.cpp

```diff
@@ -568,6 +568,7 @@ void FurnaceGUI::drawFindReplace() {
                     makeCursorUndo();
                     e->changeSongP(i.subsong);
                     if (e->isPlaying()) {
+                      wasFollowing=followPattern;
```

Comment:
Check whether followPattern is on - the user may actually have it disabled before going to a result